scripts.d/support/install-setup

```diff
@@ -105,6 +105,19 @@ cp servers.pvsc $USER_SERVER_PVSC
 
 sed -i "s/NNI_USER/$USER/g" $USER_SERVER_PVSC
 
+#  remove pyhello et hello from salome app resource file
+
```

Comment:
This could be rewritten as:

`sed -i -E '/^.*name="PYHELLO" value="/d;/^.*name="HELLO" value="/d";s/PYHELLO;//g;s/HELLO;//g' /home/$USER/.config/salome/SalomeApprc*`

Note that using single quote for the sed parameter avoid using `\"` in the pattern.

Also I don't think the last two regex are safe, that is replacing `HELLO;` by `""` could match some other parts in the file and more importantly this won't match the last entry in the `user_modules` list.

So I propose for the last two regex:

`'s/\(<parameter.*\)[";]HELLO/\1/'`
and

`'s/\(<parameter.*\)[";]PYHELLO/\1/'`